src/containers/blocks.jsx

```diff
@@ -77,6 +91,14 @@ class Blocks extends React.Component {
         this.detachVM();
         this.workspace.dispose();
     }
+    setToolboxSelectedItemByName (name) {
+        const categories = this.workspace.toolbox_.categoryMenu_.categories_;
```

Comment:
Could you add a `@todo` here too for enhancing scratch-blocks to allow this kind of thing without digging around? I know something very similar was being added to scratch-blocks for the "all-in-one" toolbox, so it would be nice to indicate that this should use that when it is available